models/CourseModel.js

```diff
@@ -1,6 +1,13 @@
 const mongoose = require('mongoose');
 const Schema   = mongoose.Schema;
 const version = require('mongoose-version');
+const isAbsoluteUrl = require('is-absolute-url');
+
+const fixUrl = (url) => {
```

Comment:
It may be helpful to rename this function to more accurately explain what it is fixing, perhaps `formatUrlFor___`.